src/Processors/QueryPlan/ReadFromMergeTree.cpp

```diff
@@ -365,7 +369,7 @@ static ActionsDAGPtr createProjection(const Block & header)
 }
 
 Pipe ReadFromMergeTree::spreadMarkRangesAmongStreamsWithOrder(
-    RangesInDataParts && parts_with_ranges,
+    RangesInDataParts parts_with_ranges,
```

Comment:
So it seems like before with `&&` we didn't need to call move constructor (and additional destructor at the end) to pass and take elements from `parts_with_ranges` and now we explicitly do. Am I missing something or how is this part better than before?